src/Model/Coordinates.php

```diff
@@ -17,7 +17,8 @@ class Coordinates
         min: -90,
         max: 90
     )]
-    private string $latitude;
+    private ?string $latitude;
```

Comment:
Czemu nullable?

Reply:
To poprawialiśmy razem na vc ale juz nie pamietam w jakim celu